apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -263,5 +268,22 @@ export default class ControlRowHeader extends mixins(HtmlSanitizeMixin) {
       return {label: cci, url: '', description: this.descriptionForTag(cci)};
     });
   }
+  showLegacy(control: any) {
+    const legacyTag = control.data.tags['legacy'];
+    if (legacyTag) {
+      if (Array.isArray(legacyTag)) {
```

Comment:
@HenryXiaoHX so this paradigm of 'thing or array of things' paradigm shows up a bunch where you do the exact same thing to both, but have to index into the array first.  so what we can do is similar to what i said with the guard clause concept, but in this case we check if something is not an array, make it an array, and now all the logic is consolidated.  it ends up making the code more DRY (Don't Repeat Yourself), which is good cause if the logic needs to be updated for some reason, you only need to make that update the once instead of twice or more (which could potentially result in you missing one of the occasions that you need to do that update).